### ISSUE TITLE
`--no-frecency` not working?

### ISSUE THREAD

**Issue by micgro42** [fdw/rofimoji#138]
I just updated to 6.0.0 and also just now added the `--no-frecency` option as the current frecrency sorting makes it almost impossible to find rarely used emojis by position when one doesn't know their keywords/description.

However, `--no-frecency` seems to plain not work? It still shows the basically scrambled list of emojis until the unused ones start.

Not sure if I have to reload anything or somesuch. I'm using rofimoji as installed via pip on Ubuntu.

PS: My preferred way forward would be to use the "frecency" functionality only for the top `n` emojis, and have all later ones be sorted positionally.

**Comment by mcourteaux** [comment]
Are you confused with the spelling of "frequency"?

**Comment by fdw** [comment]
Yes, `--no-frecency` was broken, unfortunately. Sorry for that!

It should be fixed on `main` now; can you please test it?

> PS: My preferred way forward would be to use the "frecency" functionality only for the top n emojis, and have all later ones be sorted positionally.

Thanks for that idea I'll need to think about this...

**Comment by micgro42** [comment]
@fdw Mh. It does work insofar as it is no longer sorted by frecency. However, it seems to now be confused about the order of the emoji blocks/sections? The block that starts with the "grinning face" 😀 only appears after scrolling about 60% of the way down.

(I'm running plain `rofimoji --no-frecency`, I think.)

**Comment by fdw** [comment]
Nope, that is the expected order: Emojis are now split into several files, which are loaded alphabetically. If you want to change the order (or only load subset of emoji), you can do so with `--files` 🙂

**Comment by micgro42** [comment]
Ah, that is quite unexpected! Also, the ordering is a bit surprising. Shouldn't the hearts be with the symbols?^^ That being said, looking at https://en.wikipedia.org/wiki/Emoji#In_Unicode, I'm not yet sure where that "canonical order" of emojis is even coming from.

**Comment by micgro42** [comment]
But I think this issue is resolved, and the following command should give me more-or-less what I expect:

```sh
rofimoji --no-frecency -f emojis_smileys_emotion emojis_people_body emojis_animals_nature emojis_food_drink emojis_activities emojis_travel_places emojis_objects emojis_symbols emojis_flags
```

**Comment by fdw** [comment]
The grouping of emojis comes from the Unicode consortium, see https://unicode.org/emoji/charts-15.0/full-emoji-list.html .
But regarding the order of each group, I thought it was nicer for the user to be able to define it themselves, because maybe they don't need flags or really like animals.

I'll close this ticket, as the original issue is done, but if you have a wish regarding the order, feel free to open another one 🙂
